src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java

```diff
@@ -67,13 +67,42 @@
  * &lt;module name="JavadocVariable"/&gt;
  * </pre>
  * <p>
+ * By default, this setting will report a violation if
+ * there is no javadoc for any scope member.
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private int a; // violation, missing javadoc for private member
+ *
+ *   &#47;**
+ *   * Some description here
+ *   *&#47;
+ *   private int b; // OK
+ *   protected int c; // violation, missing javadoc for protected member
+ *   public int d; // violation, missing javadoc for public member
```

Comment:
` *   /*package*/ int e; // OK`
in all exmaples.
comment is used to visually show that no visibility modifier mean "package" visibility.

@Abhishek-kumar09 points to the same problem

Reply:
@romani

**This package protected member is falling in the category of private scope**
```
λ cat Test.java
public class Test {
        private int a; // violation, missing javadoc for private member

        /**
         * Some description here
         */
        private int b; // OK
        protected int c; // OK
        public int d; // OK
        /*package*/ int e; // OK
}

λ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
        <module name="TreeWalker">
                <module name="JavadocVariable">
                        <property name="scope" value="private"/>
                        <property name="excludeScope" value="protected"/>
                </module>
        </module>
</module>

λ java -jar checkstyle-8.41-all.jar -c config.xml Test.java
Starting audit...
[ERROR] E:\GitHub\Test.java:2:9: Missing a Javadoc comment. [JavadocVariable]
[ERROR] E:\GitHub\Test.java:10:21: Missing a Javadoc comment. [JavadocVariable]
Audit done.
Checkstyle ends with 2 errors.
```

Reply:
done